### ISSUE TITLE
horizontal scroll not working when using infinite scroll

### ISSUE THREAD

**Issue by pedram6195** [kutlugsahin/react-smooth-dnd#90]
I have this usecase that there are several columns (containers) and inside them are draggable cards. each column must have a fixed `height` and `overflow-y: auto` because cards' data is coming from an external API and may be too many, so I have to implement infinite scroll inside each column. 

so the problem is when I use `overflow-y: auto` inside each column (the `div.card-container` or `div.smooth-dnd-container.vertical` no difference), and then try to drag a card from column 1 to column 8 for example, the entire container is not scrolling by itself. I've used the first example in the documentation and tweaked it a little bit to show you the problem: 
[codesandbox demo](https://codesandbox.io/s/gallant-lake-z310ud?file=/src/App.js)

[screen-recorder-wed-jul-27-2022-10-40-43.webm](https://user-images.githubusercontent.com/18122126/181174230-9fc021d4-2e2a-4b1d-8eba-637e1f1d9faf.webm)

**Comment by LeeManh** [comment]
I also have the same situation

**Comment by fredmanxu** [comment]
@pedram6195 
I'm having the same issue, do you have a solution?

**Comment by fredmanxu** [comment]
Initially I used [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd) library and it had the same problem like [here](https://github.com/atlassian/react-beautiful-dnd/issues/1783).
Finally I used the solution [here](https://github.com/atlassian/react-beautiful-dnd/issues/1783#issuecomment-690919987) and [here](https://github.com/kutlugsahin/smooth-dnd/issues/54). It worked for me.

In a nutshell
```
npm i @richardrout/react-smooth-dnd
```
Set `disableScrollOverlapDetection` prop to true.
```
import { Container } from '@richardrout/react-smooth-dnd'

return <Container
                disableScrollOverlapDetection={true}
                ...>
                 // Your lists
            </Container>
```

**Comment by Monu18765** [comment]
Thanks @fredmanxu your solution fixed the issue.
